data/tabular/peptides_soluble/meta.yaml

```diff
@@ -20,7 +20,7 @@ targets:
 benchmarks: []
 identifiers:
     - id: sequence
-      type: AS_SEQUENCE
+      type: Other
```

Comment:
And AS_SEQUENCE is needed to have a correct split, but this is fixed in the train_test_split PR